### DISCUSSION TITLE
For support and help with using or building RocketChat post on open.rocket.chat or forums.rocket.chat     For bugs only create a github issue

### DISCUSSION THREAD

**Discussion by casalsgh** [RocketChat/Rocket.Chat#34157] [General]
For support and help with using or building RocketChat post on [open.rocket.chat](https://open.rocket.chat/#) or [forums.rocket.chat](https://open.rocket.chat/#)
For bugs only create a github issue

**Comment by reetp** [comment]
You can also read all the docs:

https://docs.rocket.chat/
https://developer.rocket.chat/docs
